# Tema sobre desarrollo basado en pruebas.

## Ejercicios

- Ejercicio 1

Para este ejercicio he ejecutado el test.py de 
[tdd-gdg](https://github.com/JJ/tdd-gdg). Su respuesta ha sido:

![Respuesta de la ejecuci贸n de test.py](/Capturas/pytest.png)

- Ejercicio 2

A continuaci贸n, podemos observar como falla la ejecuci贸n del test en un assert:

![Test fallido](/Capturas/failure_test.png)

La correci贸n del mismo ha sido la siguiente:

![Test corregido](/Capturas/test.png)
